recipe/meta.yaml

```diff
@@ -91,28 +101,37 @@ outputs:
         - cuda-version >=11.0.*,<12.0.a0  # [(cuda_compiler_version or "").startswith("11")]
       run_constrained:
         - cuda-version >=11.0.*,<13.a0    # [cuda_compiler_version == "None"]
-        - mkl-devel 2024.4
+        - mkl 2024
         - libnvpl-fft0 >=0.3.0,<1.0a0
+        - mathdx 24.04.0.*
+        - numba 0.60.*
+        # run_constrained because this is an off-channel package
+        - pynvjitlink >=0.2
     test:
+      requires:
+        - pip
       imports:
         - nvmath
+        - nvmath.fft
+        - nvmath.linalg
+      commands:
+        - pip check
 
   - name: nvmath-python-dx
-    version: {{ version }}
     build:
-      number: {{ number }}
-      skip: true  # [py < 310 or py > 312]
-      skip: true  # [not (linux64 or aarch64)]
+      noarch: generic
+      force_ignore_keys:
+        - cuda_compiler_version
+      # There will be only one variant of this meta-package for all platforms, but we want
+      # to run the tests with multiple variants of the base package.
+      # dx only supports linux right now
       # dx support currently requires CUDA 12
-      skip: true  # [not (cuda_compiler_version or "").startswith("12")]
-      script:
-        - echo "building the [dx] meta package..."
+      skip: true  # [(not linux) or (not (cuda_compiler_version or "").startswith("12"))]
     requirements:
-      host:
-        - python
       run:
-        - python
-        - {{ pin_subpackage("nvmath-python", exact=True) }}
+        # TODO: Move versions to run_constraints on the main package in next release.
+        - __linux
+        - {{ pin_subpackage("nvmath-python", max_pin='x.x.x') }}
```

Comment:
Is this pinning correct? Can the `dx` package rely on a non-cuda build variant? Why do we have non-cuda build variant?

Reply:
I think no, so I'll need to update the recipe.